script/test

```diff
@@ -1,3 +1,16 @@
-#!/bin/bash
-set -ex
-npm run check
+#!/usr/bin/env node
```

Comment:
Should this attempt to install dependencies for sub-projects?  I have to run `npm install` inside `generate-eslint-shopify` to get this working (otherwise, it spits out `Uncaught Error: Cannot find module 'yosay'`).